components/polylith/yaml/core.py

```diff
@@ -0,0 +1,12 @@
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
```

Comment:
MyPy suggests types-PyYAML to be installed, when running `poetry run mypy .`

Maybe adding it as a dev-dependency? `poetry add types-PyYAML --group dev`

Reply:
Done in 9435c2f